pkg/codegen/testing/test/testdata/azure-native-nested-types/docs/documentdb/sqlresourcesqlcontainer/_index.md

```diff
@@ -837,9 +837,11 @@ All [input](#inputs) properties are implicitly available as output properties. A
 </dd></dl>
 </pulumi-choosable>
 </div>
+
 ## Import
 
 
+
```

Comment:
?

Reply:
This is test data generated as a result of https://github.com/pulumi/pulumi/pull/9501/files#diff-9dd78c1c67e7da05e337d154a72300056952324ad91ada1df19409c69ccbc5e2.